SDK/MobileCenter/Microsoft.Azure.Mobile.Android/Resources/Resource.Designer.cs

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Delete this file, it's supposed to git ignored anyway, it's like you force added it.